apis/v1alpha1/nginxproxy_types.go

```diff
@@ -10,7 +10,7 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 // NginxProxy is a configuration object that is attached to a GatewayClass parametersRef. It provides a way
 // to configure global settings for all Gateways defined from the GatewayClass.
-type NginxProxy struct { //nolint:govet // standard field alignment, don't change it
```

Comment:
The golangci-lint-root keeps removing this comment when I commit. I tried moving above the function definition but does the same.

Is it still required to have standard field alignment for this struct?

Reply:
No, I wouldn't worry about alignment here. I don't think the pipeline complains about it.